integration/use-cases/operator-deployment.js

```diff
@@ -37,24 +47,31 @@ test("Deploys an Operator", async () => {
     await expect(page).toMatch("Operators", { timeout: 10000 });
 
     // Filter out charts to search only for the prometheus operator
-    await expect(page).toClick("label", { text: "Operators" });
+    await expect(page).toClick("label", { text: "Operators", timeout: 10000 });
```

Comment:
the only `timeout` option working is for the `toMatch` function:

https://github.com/smooth-code/jest-puppeteer/blob/master/packages/expect-puppeteer/README.md#expectinstancetomatchmatcher-options